### DISCUSSION TITLE
is there any way to change the size of the thumnails?

### DISCUSSION THREAD

**Discussion by nawta** [LekoArts/gatsby-themes#939] [Questions] [answered]
I use this great gatsby-starter-portfolio-emilia(https://github.com/LekoArts/gatsby-starter-portfolio-emilia) for my portfolio site(https://nawta.github.io/), and it's really amazing theme. I love it. However, I think it's way better to make the thumbnails of the work(example pictures below) smaller, is there any way to change that?
<img width="1408" alt="スクリーンショット 2022-07-12 8 34 13" src="https://user-images.githubusercontent.com/39507181/178375912-45a79825-e673-4065-ab0c-8184100ca5f7.png">
<img width="1125" alt="スクリーンショット 2022-07-12 8 36 07" src="https://user-images.githubusercontent.com/39507181/178376095-4e70fd91-32a7-4319-be53-16eec44d4791.png">

**Comment by LekoArts** [comment, marked as answer]
You can shadow the `projects` component do change the CSS grid layout: https://github.com/LekoArts/gatsby-themes/blob/main/themes/gatsby-theme-emilia/src/components/projects.tsx#L75

**Reply by nawta** [reply]
Thank you! That helped me a lot
